test/pytest/test_handler.py

```diff
@@ -224,7 +224,28 @@ def test_kfserving_mnist_model_register_and_inference_on_valid_model_explain():
 
     assert np.array(json.loads(response.content)['explanations']).shape == (1, 1, 28, 28)
     test_utils.unregister_model("mnist")
+
+def test_huggingface_bert_batch_inference():
+    batch_size = 2
+    batch_delay = 10000 # 10 seconds
+    params = (
+        ('model_name', 'BERTSeqClassification'),
+        ('url', 'https://bert-mar-file.s3.us-west-2.amazonaws.com/BERTSeqClassification.mar'),
```

Comment:
@msaroufim We need to change the url to mar file to the one in Torchserve s3 bucket

Reply:
Fixed